mod_app/admin/link_admin.py

```diff
@@ -14,7 +14,7 @@ class SourceInline(admin.TabularInline):
     ]
 
 
-class VideoInline(admin.TabularInline):
+class VideoInline(admin.TabularInline, PreviewMixin):
```

Comment:
See below for why, but generally best to [put Mixins first](https://realpython.com/inheritance-composition-python/#inheriting-multiple-classes).
```suggestion
class VideoInline(PreviewMixin, admin.TabularInline):
```